### ISSUE TITLE
internal issue error

### ISSUE THREAD

**Issue by piyush-mahajan** [adrianhajdin/project_openai_codex#41]
POST http://localhost:5000/ 500 (Internal Server Error)

**Comment by ArslanYM** [comment]
I'm getting the same error!

**Comment by iqlipse-abhi** [comment]
You guys got any solution?

**Comment by ArslanYM** [comment]
Guys , i was having the same issue till yesterday. I fixed it today, firstly make all the changes he is making in the tutorial and then change the backend server from port 5000 to 3000 or some other and change it in script.js as well as server.js then clear both the terminals and run it again. It works fine.

**Comment by ArslanYM** [comment]
My code is here : https://github.com/ArslanYM/Chat-GPT-Clone . Let me know it this helps

**Comment by sourabh-xgod** [comment]
its replying "something went wrong" @ArslanYM

**Comment by iqlipse-abhi** [comment]
I just deleted my api key from open ai generated a new one copy pasted the same in .env and boom it started working

**Comment by sourabh-xgod** [comment]
okay will try that

**Comment by ArslanYM** [comment]
> its replying "something went wrong" @ArslanYM

yeah there is some issue with running it on render but the code works well on local host if you do the above mentioned steps
